pluggy.py

```diff
@@ -412,14 +423,16 @@ def _verify_hook(self, hook, hookimpl):
                 "Plugin %r\nhook %r\nhistoric incompatible to hookwrapper" %
                 (hookimpl.plugin_name, hook.name))
 
-        for arg in hookimpl.argnames:
-            if arg not in hook.argnames:
-                raise PluginValidationError(
-                    "Plugin %r\nhook %r\nargument %r not available\n"
-                    "plugin definition: %s\n"
-                    "available hookargs: %s" %
-                    (hookimpl.plugin_name, hook.name, arg,
-                     _formatdef(hookimpl.function), ", ".join(hook.argnames)))
+        # positional arg checking
+        notinspec = set(hookimpl.argnames) - set(hook.argnames)
+        if notinspec:
+            raise PluginValidationError(
+                "Plugin %r for hook %r\nhookimpl definition: %s\n"
```

Comment:
@nicoddemus no you're definitions are absolutely correct.
Sorry when I said "keyword" I meant to say "default".

So yes my concern is about being explicit with regard to `hookimpl` to `hookspec` mismatches regarding **default** arguments which is really not in the scope of this PR (it's coupled here because originally #43 and this were one PR).

I'll revert and thanks for clarifying.